sdk/python/lib/pulumi/config.py

```diff
@@ -66,21 +66,28 @@ def _get(
         #         f"use `{use.__name__}` instead of `{instead_of.__name__}`")
         return get_config(full_key)
 
-    def get(self, key: str) -> Optional[str]:
+    def get(self, key: str, default: Optional[str] = None) -> Optional[str]:
         """
-        Returns an optional configuration value by its key, or None if it doesn't exist.
+        Returns an optional configuration value by its key,
+        a default value if that key is unset and a default is provided,
+        or None if it doesn't exist.
 
         :param str key: The requested configuration key.
+        :param Optional[Any] default: An optional fallback value to use if the given configuration key is not set.
         :return: The configuration key's value, or None if one does not exist.
         :rtype: Optional[str]
         """
-        return self._get(key, self.get_secret, self.get)
+        config_candidate = self._get(key, self.get_secret, self.get)
+        return config_candidate if config_candidate is not None else default
 
     def get_secret(self, key: str) -> Optional[Output[str]]:
```

Comment:
Where is the default parameter? I see it as a `:param` in the comments but not in the function signature.

Reply:
Hey @iwahbe !  It's on line 69 here - `default: Optional[str] = None`